content/documentation/Ox64/Software/Flashing.adoc

```diff
@@ -350,14 +365,14 @@ $ sudo dd if=sdcard.img of=/dev/[DEVICE] bs=1M status=progress conv=fsync
 
 Power off your Ox64 and insert the microSD card.
 
-Open a terminal window to connect to the D0 core’s (i.e. Linux’s) serial console:
+Open a terminal window to connect to the D0 core’s (i.e. Linux’s) serial console. If you are using a Pico or Bluepill as your serial adapter, this will be `/dev/ttyACM0`. If you are using a generic serial adapter, use the same `/dev/tty[DEVICE]` file as you did in the flashing section:
 
 [source,console]
 ----
-$ minicom -b 2000000 -D /dev/ttyACM0
+$ minicom -b 2000000 -D /dev/tty[DEVICE]
```

Comment:
I was not able to solve the clarification as you propose in this PR, but in genreal you did a great job. But regarding ttyACM0 and ttyACM1 is really important to make clear there are 2 ports linked to different cores of ox64.